cloud/.env.example

```
@@ -0,0 +1,9 @@
+# Supabase (from supabase.com dashboard -> Settings -> API)
+GRADATA_SUPABASE_URL=https://YOURS.supabase.co
+GRADATA_SUPABASE_ANON_KEY=YOURS
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRADATA_SUPABASE_ANON_KEY key should go before the GRADATA_SUPABASE_URL key

(UnorderedKey)